new-lamassu-admin/src/pages/Machines/MachineComponents/Transactions/CopyToClipboard.js

```diff
@@ -0,0 +1,74 @@
+import { makeStyles } from '@material-ui/core/styles'
```

Comment:
wasn't this removed? I think this is true for all of the changes in the `Transactions` folder